# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1310855108 25200
#      Sat Jul 16 15:25:08 2011 -0700
# Node ID 29d4b27e485dec11c2effe1b7479a1002a359964
# Parent  e51ad94535b9654d0fc2e529b2df92ceaf5d4321
Remove warning_ids.m from test statistics.

* warning_ids.m: Add %!assert(1) to remove from list of untested functions.

diff --git a/scripts/miscellaneous/warning_ids.m b/scripts/miscellaneous/warning_ids.m
--- a/scripts/miscellaneous/warning_ids.m
+++ b/scripts/miscellaneous/warning_ids.m
@@ -279,8 +279,11 @@
 ## will print a warning if a switch label is not a constant or constant
 ## expression.  By default, the @code{Octave:variable-switch-label}
 ## warning is disabled.
 ## @end table
 
 function warning_ids ()
   help ("warning_ids");
 endfunction
+
+## Remove from test statistics.  No real tests possible
+%!assert (1)
